values-schema.yaml

```diff
@@ -63,6 +63,7 @@ definitions:
     $ref: '#/definitions/labelsAnnotations'
     description: A set of annotations.
   azureMonitor:
+    readOnly: false
```

Comment:
Again, I have no clue why this property is added. Maybe it wasn't you and some weird merging is happening, but I never saw it before and it should not be here I think.

Reply:
You're right, I don't know what this is doing here. I'll remove it.